docs/dspl2-spec.md

```diff
@@ -60,6 +66,15 @@ We introduce the following constructs:
 
 While most of the constructs above describe "metadata" about the dataset, we also need to represent actual data. The preferred way of providing this data is to use CSV tables. An equivalent triple representation is also supported.
 
+### Notes
+#### Provenance
+
+Many statistical datasets include data from multiple providers. The [CreativeWork](http://schema.org/CreativeWork) superclass for StatisticalDataset offers support for multiple authors and publishers at the dataset level, but does not address provenance for individual measures or timeseries observations.
```

Comment:
Maybe we should add a sentence to say that we stick with that for now, but may add support for provenance at a finer granularity in the future?